tutorials/plot_background_freesurfer.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+r"""
+===============================
+About MNE-Python and FreeSurfer
+===============================
+
+FreeSurfer is an open source analysis toolbox for MRI data. It contains several
+command line tools and graphical user interfaces. FreeSurfer can be obtained
+from https://surfer.nmr.mgh.harvard.edu/
+
+In MNE-Python, FreeSurfer is used to provide structural information of various
+kinds, for :ref:`source estimation
+<sphx_glr_auto_tutorials_plot_mne_dspm_source_localization.py>`. Thereby a
+subject specific structural MRI will be transformed into various
```

Comment:
MRI is not transformed to "spherical or inflated brain surfaces".
Maybe
"spherical or inflated brain surfaces are obtained from MRI anatomical images"